packages/eas-cli/src/commands/metadata/configure.ts

```diff
@@ -0,0 +1,86 @@
+import { getConfig } from '@expo/config';
+import { Errors, Flags } from '@oclif/core';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import Log, { learnMore } from '../../log';
+import { createMetadataContextAsync } from '../../metadata/context';
+import { downloadMetadataAsync } from '../../metadata/download';
+import { handleMetadataError } from '../../metadata/errors';
+import { RequestedPlatform } from '../../platform';
+import { findProjectRootAsync, getProjectIdAsync } from '../../project/projectUtils';
+
+type RawCommandFlags = {
+  platform?: string;
+  profile?: string;
+  'non-interactive': boolean;
+};
+
+type CommandFlags = {
+  requestedPlatforms: RequestedPlatform;
+  profile?: string;
+  nonInteractive: boolean;
+};
+
+export default class MetadataConfigure extends EasCommand {
+  static description = 'upload metadata configuration to the app stores';
+
+  static flags = {
+    platform: Flags.enum({
+      char: 'p',
+      options: ['ios'],
+    }),
+    profile: Flags.string({
+      description:
+        'Name of the submit profile from eas.json. Defaults to "production" if defined in eas.json.',
+    }),
+    'non-interactive': Flags.boolean({
+      default: false,
+      description: 'Run command in non-interactive mode',
+    }),
+  };
+
+  static args = [];
+
+  async runAsync(): Promise<void> {
+    const { flags: rawFlags } = await this.parse(MetadataConfigure);
+    const flags = await this.sanitizeFlagsAsync(rawFlags);
+
+    const projectDir = await findProjectRootAsync();
+    const { exp } = getConfig(projectDir, { skipSDKVersionRequirement: true });
+    await getProjectIdAsync(exp);
+
+    const metadataContext = await createMetadataContextAsync({
+      projectDir,
+      exp,
+      profileName: flags.profile,
+      nonInteractive: flags.nonInteractive,
+    });
+
+    try {
+      Log.addNewLineIfNone();
+      const filePath = await downloadMetadataAsync(metadataContext);
+      Log.addNewLineIfNone();
+
+      Log.succeed('Your store configuration has been generated!');
+      Log.log(filePath);
+      Log.log(learnMore('https://docs.expo.dev/eas-metadata/introduction/'));
```

Comment:
This is not available yet, before merging I have to check if we have this ready or if we should omit this link until the docs are ready.